pyproj/utils.py

```diff
@@ -151,3 +152,40 @@ def _convertback(data_type: DataType, inx: Any) -> Any:
     if data_type == DataType.TUPLE:
         return tuple(inx)
     return inx
+
+
+def reverse_azimuth(azi: float, radians: bool = False) -> float:
```

Comment:
I would say these functions are a better fit for `geod.py` than a generic pyproj utility.

Reply:
It looks like they are only used for testing. Thoughts about moving them to the testing module instead?

Reply:
It may be more performant to move them to `_geod.pyx` and import into `_geod.py`.

Reply:
> It may be more performant to move them to `_geod.pyx` and import into `geod.py`.

makes sense, would you mind doing that after this merge as I don't have cython installed and I don't have more time to invest in this...